src/deprecated/ChoiceInputField.tsx

```diff
@@ -25,6 +25,9 @@ const getInputToRender = (inputType: 'radio' | 'checkbox', children?: React.Reac
   )
 }
 
+/**
+ * @deprecated Use `FormControl` instead. See https://primer.style/react/FormControl for more info
```

Comment:
Have you verified this annotation works? I think it'll need to be applied at the bottom of the file in the object assignment instead.